contracts/experimental/ERC20ProxyForZRC2/test/basicProxyTest.ts

```diff
@@ -0,0 +1,179 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+import hre from "hardhat";
+import { ScillaContract } from "hardhat-scilla-plugin";
+import { Account } from "@zilliqa-js/zilliqa";
+import { initZilliqa } from "hardhat-scilla-plugin";
+
+/// not.to.be.reverted seems to not work here (it complains about null receipts), so ..
+describe("basicTest", function () {
+  let zrc2Contract: ScillaContract;
+  let erc20Proxy: Contract;
+  let zrc2Owner: Account;
+
+  //let zrc2OwnerAddress: string;
+
+  let zrc2OwnerEVM: Signer;
```

Comment:
```suggestion
  let zrc2OwnerEVM: Wallet;
```

Reply:
The type updates makes most of the IDE red lines goes away. Still there are a lot of lines about `| undefined` issue, but I guess there is only so much we can fix, so let's it be for now.

Reply:
This takes us back to the messiness of TS/JS ecosystem. I bet you won't say we can't make every IDE report the same Rust code as correct.
There are tools for TS to improve code quality, like clippy does for Rust. We are even using them, but not in this repo. Maybe one day.